src/components/contextual/pages/pool/PoolActionsCard.vue

```diff
@@ -36,6 +37,7 @@ const { networkSlug } = useNetwork();
  */
 const joinDisabled = computed(
   (): boolean =>
+    POOLS.Deprecated?.[props.pool.id].joinsDisabled ||
```

Comment:
Maybe makes sense to have a helper function for this in usePool?
```suggestion
    deprecatedDetails(props.pool.id)?.joinsDisabled ||
```